src/new-client/src/App/Analytics/Analytics.tsx

```diff
@@ -17,7 +17,9 @@ const AnalyticsWrapper = styled.div`
 
 analytics$.subscribe()
 
-const loader = <Loader minWidth="22rem" minHeight="22rem" />
+const loader = (
+  <Loader ariaLabel="analytics-loader" minWidth="22rem" minHeight="22rem" />
```

Comment:
The `aria-label` attribute is not meant to be used like that. IMO this makes the App less accessible.

> The aria-label attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen.

This is text that an assistant reader would read to the user. However, that text doesn't really mean anything to anyone but to our tests. Ideally, the loader should have some sort alt text like "Loading..." and then we could use that text to target the Loader that's specific to a particular test. Otherwise, let's use the `getByTestId` query.

Same thing for all the other miss usages.

Reply:
I agree, let's ensure that aria-* attributes are used for accessibility. Getting accessibility right is a big story that we've not really covered before, so I don't want to conflate that with this test work. Let's use a test-id here instead.